packages/client/src/Components/Mapping/helpers/MapAdjudicatingListener.js

```diff
@@ -1,5 +1,5 @@
 import L from 'leaflet'
-import defaultHexStyle from './data/default-hex-style'
+import defaultHexStyle from '../../../Helpers/data/default-hex-style'
```

Comment:
@foxleigh81 - I guess `defaultHexStyle` would also go into the mapping helpers package, since it only relates to the mapping component.